package.json

```diff
@@ -48,6 +48,7 @@
     "js2xmlparser": "^3.0.0",
     "json2csv": "^4.1.2",
     "jwt-decode": "^2.2.0",
+    "loglevel": "^1.7.1",
```

Comment:
Do we need a dependency for this? Will we use some advanced feature that the package offers and we don't want to develop?